qcelemental/tests/test_model_results.py

```diff
@@ -1062,6 +1104,64 @@ def test_model_survey_dictable(smodel1, smodel2, every_model_fixture, request, s
     assert instance
 
 
+@pytest.mark.parametrize("smodel1,smodel2", _model_classes_struct)
+def test_model_survey_convertable(smodel1, smodel2, every_model_fixture, request, schema_versions):
+    anskey = request.node.callspec.id.replace("None", "v1")
+    # fmt: off
+    ans = [
```

Comment:
Nit: Consider making this a set given that it seems to only be used for lookup and is not used with indexes.